playbooks/roles/certbot/tasks/main.yml

```diff
@@ -29,14 +29,23 @@
     port: 80
     proto: tcp
 
-- name: add certbot PPA
-  apt_repository:
-    repo: ppa:certbot/certbot
+# Certbot apt packages are outdated and no longer officially supported by certbot.
+# The oficially supported installation method is via snap.
+- name: Remove certbot apt package if installed
```

Comment:
@mtyaka, it looks like this doesn't remove the dummy cronjob that the `certbot` package installs. While the cronjob itself will do nothing, it will be a good idea to remove it to avoid any confusion.

Reply:
Good idea 👍 I'll do it manually.

Reply:
Manually removed the old cronjob from all servers to which we already installed the new version.

Reply:
@mtyaka, the `certbot` package hasn't been removed properly on one of our servers and I have mentioned the details in my reply on the task. Can you check that?

Reply:
Thanks for catching that... I'm not sure what happened, but I do remember that some ansible tasks were taking unusually long to complete on that server when I ran the playbooks yesterday. Maybe 

I re-ran the playbook today and it was stuck on the "Remove certbot apt package if installed" for almost 20 minutes. However after 20 minutes it successfully completed and I verified that the certbot apt package is no longer installed on the server.